### DISCUSSION TITLE
My website is almost ready: which Speed and SEO analyzer do you recommend?

### DISCUSSION THREAD

**Discussion by Vinc26** [GetPublii/Publii#1849] [General]
Hi,

It's there: [https://aurelielamour.com/](https://aurelielamour.com/).

I would now like to see if everything is going well, or what I should still improve in terms of speed and SEO. Which offline or online analyzer do you recommend? If possible not too expensive 😬

🙏

**Comment by michaelfluegge** [comment]
I googled it for you and found: https://forum.getpublii.com/topic/summary-seo-guidelines/

**Reply by Vinc26** [reply]
Thanks!

**Reply by Somalilandpower** [reply]
test on google PageSpeed Insights https://pagespeed.web.dev/analysis/https-aurelielamour-com/lwvsdo0vk9?form_factor=desktop Good Luck

**Comment by candidexmedia** [comment]
I like these ones:
- https://yellowlab.tools/
- https://ecograder.com/
- https://www.w3.org/WAI/test-evaluate/tools/list/

**Reply by Vinc26** [reply]
Thanks!
